pkg/util/parameters.go

```diff
@@ -0,0 +1,180 @@
+package util
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	ackerr "github.com/aws-controllers-k8s/runtime/pkg/errors"
+)
+
+var (
+	errUnknownParameter      = fmt.Errorf("unknown parameter")
+	errUnmodifiableParameter = fmt.Errorf("parameter is not modifiable")
```

Comment:
nit: i know those errors are not imported anywhere, but since they are pre-defined errors they need to be exposed (public) in case in one wants to select types on them.

Reply:
Done